pnpm-lock.yaml

```diff
@@ -1,5 +1,9 @@
 lockfileVersion: '6.0'
```

Comment:
One thing I'm curious about and am having a hard time finding after ~5mins of googling is: if we completely delete the `pnpm-lock` file here and re-reun `pnpm i` will this be updated to the latest version (6.1)?  Do we care about that? I'm not sure so thought I'd ask!

Reply:
Thanks for asking! I was wondering the same thing. Deleting `pnpm-lock.yaml` and running `pnpm i` creates a lockfile diff for some reason but doesn't bump the lockfile version to 6.1.

From PNPM's release notes, it looks like they bumped the lockfile to 6.1 with PNPM [8.6.0](https://github.com/pnpm/pnpm/releases/tag/v8.6.0) but then downgraded the lockfile back down to 6.0 with PNPM [8.6.2](https://github.com/pnpm/pnpm/releases/tag/v8.6.2). Perhaps that explains why we're not getting a 6.1 lockfile.

Reply:
Ah, interesting! Thanks for digging in.